src/personas/maid_manager/tests.rs

```diff
@@ -80,6 +81,9 @@ fn idata_basics() {
     assert_eq!(message.dst, client);
     assert_match!(message.response, Response::PutIData { res: Ok(()), .. });
 
+    // Simulate refresh from at least QUORUM nodes.
+    simulate_refresh(&mut node, &mut mm, msg_id, QUORUM);
```

Comment:
That shouldn't be necessary anymore, now that we update the count on `PutSuccess`? (Also below.)